code/modules/mob/living/carbon/human/species_types/humans.dm

```diff
@@ -7,6 +7,8 @@
 	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
 	use_skintones = 1
 	skinned_type = /obj/item/stack/sheet/animalhide/human
+	disliked_food = GROSS | RAW

```

Comment:
i hate carpaccio so it might as well be :^) either way you can just remove the raw flag from anything that you consider edible raw. I just did it on most things that i consider not very great when eaten raw by a civilized species.